README.md

```diff
@@ -847,9 +874,24 @@ braker.pl --UTR=on --softmasking --genome=genome.fa --bam=rnaseq.bam \
 
 Modify `augustus.hints.gtf` to point to the AUGUSTUS predictions with hints from previous BRAKER run; modify flaning_DNA value to the flanking region from the log file of your previous BRAKER run; modify some_new_working_directory to the location where BRAKER should store results of the additional BRAKER run; modify somespecies to the species name used in your previous BRAKER run.
 
+### --addUTR=on
+
+Add UTRs from RNA-Seq converage information to AUGUSTUS gene predictions using GUSHR. No training of UTR parameters and no gene prediction with UTR parameters is performed.
+
+If you performed a BRAKER run without --addUTR=on, you can add UTRs results of a previous BRAKER run with the following command:
+
+```
+braker.pl --addUTR=on --softmasking --genome=genome.fa --bam=rnaseq.bam \
+   --workingdir=some_new_working_directory \
+   --AUGUSTUS_hints_preds=augustus.hints.gtf
+   --species=somespecies --useexisting
+```
```

Comment:
We could also add a test illustrating a run in this mode. It could be for example `test3_add_utr.sh` to show how to add UTRs to an existing run in etp mode.

Reply:
Great idea. I added such a script to the tests and results folders and updated the README accordingly. Also debugged gff3 conversion of GUSHR output from BRAKER.